tests/test_storage.py

```diff
@@ -0,0 +1,48 @@
+import os
```

Comment:
We should move all the tests from `test_storage.py`

Reply:
I didn't know that file existed, I just created this one. I'll move all existing tests 🙂